air/src/two_row_matrix.rs

```diff
@@ -1,7 +1,6 @@
-use core::iter::Cloned;
-use core::slice;
+use core::{iter, slice};
 
-use p3_matrix::{Matrix, MatrixRowSlices, MatrixRows};
+use p3_matrix::Matrix;
 
 #[derive(Copy, Clone, Debug)]
 pub struct TwoRowMatrixView<'a, T> {
```

Comment:
Should we impl `row_slice`, to avoid the default which allocates? (Or maybe not after optimizations)

Reply:
Good catch